spec/features/ingredient_search_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
+
+RSpec.describe 'searching using ingredients' do
+  describe '/' do
+    it 'takes a search term' do
+      VCR.use_cassette 'sweet_tato' do
+        visit '/'
+
+        fill_in('q', with: 'sweet potatoes')
+        click_on('Search')
+
+        expect(page).to have_current_path('/foods', ignore_query: true)
+      end
+    end
+  end
+  describe '/foods' do
+    describe 'search sweet potatoes' do
+      it 'displays search results' do
+        VCR.use_cassette 'sweet_tato' do
+          visit '/'
+  
+          fill_in('q', with: 'sweet potatoes')
+          click_on('Search')
+  
+          expect(page).to have_content('Total Results: 45,635')
+          expect(page).to have_content('Food:', count: 10)
+          expect(page).to have_content('GTIN/UPC:', count: 10)
+          expect(page).to have_content('Description:', count: 10)
+          expect(page).to have_content('Brand Owner:', count: 10)
+          expect(page).to have_content('Ingredients:', count: 10)
+          expect(page).to have_content('ARCHER FARMS')
+          expect(page).to have_content('492111402857')
+          expect(page).to have_content('SWEET POTATOES')
```

Comment:
Nice work, here, you can also test to make sure a certain class appears on the page a certain amount of times.